### PR TITLE
Add kubernetes manifest for CodeGen

### PR DESCRIPTION
**Pull request by yongfengdu**
Deploy the codegen by kubectl apply.

## Description

Add manifest to deploy with kubectl.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds new functionality)

### CODE REVIEW

**Review by jfding — COMMENTED** [review]
suggest to remove unused code instead of commenting out

File: manifests/CodeGen/xeon/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+You should have all the images
+
+- tgi: ghcr.io/huggingface/text-generation-inference:1.4
+- llm-uservice: opea/gen-ai-comps:llm-tgi-server
+- codegen: opea/gen-ai-comps:codegen-megaservice-server
+
+> [NOTE]  
+> Please refer this OPEA repo https://github.com/opea-project/GenAIExamples/blob/main/CodeGen/docker-composer/xeon/README.md to build some opea images
+
```
Comment (daisy-ycguo) [inline manifests/CodeGen/xeon/README.md:13]: We don't need prebuild these images. These images will be ready in our first release.

**Review by daisy-ycguo — CHANGES_REQUESTED** [review]

**Review by daisy-ycguo — APPROVED** [review]
LGTM

**Review by lianhao — APPROVED** [review]
LGTM

**Pull request merged**